### PR TITLE
Feature/MZ-299

### PR DESCRIPTION
**Pull request by andrerussowsky**
# Midaz Pull Request Checklist

## Pull Request Type
[//]: # (Check the appropriate box for the type of pull request.)

- [ ] Ledger
- [ ] Auth
- [ ] Mdz
- [x] Transaction
- [ ] Pipeline
- [ ] Documentation

## Checklist
Please check each item after it's completed.

- [x] I have tested these changes locally.
- [ ] I have updated the documentation accordingly.
- [x] I have added necessary comments to the code, especially in complex areas.
- [x] I have ensured that my changes adhere to the project's coding standards.
- [x] I have checked for any potential security issues.
- [x] I have ensured that all tests pass.
- [ ] I have updated the version appropriately (if applicable).
- [x] I have confirmed this code is ready for review.

## Additional Notes
[//]: # (Add any additional notes, context, or explanation that could be helpful for reviewers.)
## Obs: Please always remember to target your PR to develop branch instead of main.

### CODE REVIEW

**Review by MartinezAvellan — APPROVED** [review]

File: components/transaction/internal/adapters/database/postgres/operation.postgresql.go [hunk 1]
```diff
@@ -328,6 +328,58 @@ func (r *OperationPostgreSQLRepository) FindByAccount(ctx context.Context, organ
...
+		&operation.PortfolioID,
+		&operation.ChartOfAccounts,
+		&operation.OrganizationID,
+		&operation.LedgerID,
+		&operation.CreatedAt,
+		&operation.UpdatedAt,
+		&operation.DeletedAt,
+	); err != nil {
+		if errors.Is(err, sql.ErrNoRows) {
+			return nil, common.EntityNotFoundError{
```
Comment (qnen) [inline components/transaction/internal/adapters/database/postgres/operation.postgresql.go:369]: Change the error return to comply with the pattern on ledger error responses.

File: postman/MIDAZ.postman_collection.json [hunk 2]
```diff
@@ -1711,6 +1834,65 @@
 						}
 					},
 					"response": []
+				},
+				{
+					"name": "Accounts by by asset id",
```
Comment (qnen) [inline postman/MIDAZ.postman_collection.json]: The postman file isn't up to date. Get the last version then add your changes, pls.

Comment (andrerussowsky) [reply postman/MIDAZ.postman_collection.json]: updated

**Review by qnen — APPROVED** [review]
lgtm.

**Pull request merged**